python/src/trezorlib/ethereum.py

```diff
@@ -357,3 +357,16 @@ def verify_message(
     except exceptions.TrezorFailure:
         return False
     return isinstance(resp, messages.Success)
+
+
+@expect(messages.EthereumTypedDataSignature)
+def sign_typed_data_hash(
+    client: "TrezorClient", n: "Address", domain_hash: AnyStr, message_hash: AnyStr
+) -> "MessageType":
+    return client.call(
+        messages.EthereumSignTypedHash(
+            address_n=n,
+            domain_separator_hash=normalize_nfc(domain_hash),
+            message_hash=normalize_nfc(message_hash),
```

Comment:
`normalize_nfc` shouldn't be used here as we're dealing with binary data, not Unicode. Probably best to declare the arguments as `bytes` instead of `AnyStr` and do hex->bytes conversion in the the caller.